apps/web/lib/hooks/useSlots.ts

```diff
@@ -56,6 +58,11 @@ export const getFilteredTimes = (props: getFilteredTimesProps) => {
       const slotStartTime = times[i];
       const slotEndTime = times[i].add(eventLength, "minutes");
       const slotStartTimeWithBeforeBuffer = times[i].subtract(beforeBufferTime, "minutes");
+      // If the event has seats then see if there is already a booking (want to show full bookings as well)
+      if (currentSeats?.some((booking) => booking.startTime === slotStartTime.toISOString())) {
+        console.log("This triggered");
+        break;
+      }
```

Comment:
This function filters time slots based on if there are events already in the user's calendar. Since we want to show time slots even if there is already a booking we pass them through, only if the slot matches a previous booking with seats.